pkgs/top-level/qt5-packages.nix

```
@@ -235,5 +235,7 @@ in (kdeFrameworks // plasmaMobileGear // plasma5 // plasma5.thirdParty // kdeGea
 
   soundkonverter = callPackage ../applications/audio/soundkonverter {};
 
+  xp-pen-deco-01-v2-driver = callPackage ../misc/drivers/xp-pen-deco-01-v2-driver {};
```

I'm not entirely up to date on the qt packaging conventions, should this be apart of both this file and `all-packages.nix`?

---

[I believe so.](https://nixos.org/manual/nixpkgs/unstable/#adding-an-application-to-nixpkgs)

---

According to that it should look like this:
```suggestion
  xp-pen-deco-01-v2-driver = libsForQt5.xp-pen-deco-01-v2-driver;
```